easybuild/easyconfigs/w/WRF/WRF-4.1.3-foss-2019b-dmpar.eb

```diff
@@ -0,0 +1,42 @@
+name = 'WRF'
+version = '4.1.3'
+
+homepage = 'https://www.wrf-model.org'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
+
+source_urls = ['https://github.com/wrf-model/WRF/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'WRF_parallel_build_fix.patch',
+    'WRFv4_netCDF-Fortran_separate_path.patch',
+]
+checksums = [
+    'a020e83e8a7a927fb080075031630e2af592e9fa802293ad72caa7ee50d6968d',  # v4.1.3.tar.gz
+    'f93bb6dbb8b52d72f816e2f9a6815bffd536afeca8511552ec5abc4253a59346',  # WRF_parallel_build_fix.patch
+    'a431159180757a895c07fc193fe93bb69c4c9d25d31adcc7c2adfa7c1edb04ad',  # WRFv4_netCDF-Fortran_separate_path.patch
+]
+
+# csh is used by WRF install scripts
+builddependencies = [
+    ('tcsh', '6.22.02'),
+    ('time', '1.9'),
+]
+
+dependencies = [
+    ('JasPer', '2.0.14'),
+    ('netCDF', '4.7.1'),
+    ('netCDF-Fortran', '4.5.2'),
+]
+
+# limit parallel build to 20
+maxparallel = 20
+
+buildtype = "dmpar"
+versionsuffix = '-%s' % buildtype
```

Comment:
@tstrempel can you please move these 2 lines to the top of the file (below version)?

Reply:
and while you're doing that, could you do the same for the intel version? WRF-4.1.3-intel-2019b-dmpar.eb